### DISCUSSION TITLE
cloudflare已经不支持通过API来更新freenom免费域名( .ga, .gq, .ml , .tk

### DISCUSSION THREAD

**Discussion by dh53128** [NewFuture/DDNS#299] [General]
freenom 免费的域名现在不支持从api更新，有大佬提示下解决办法么
You cannot use this API for domains with a .cf, .ga, .gq, .ml,
 or .tk TLD (top-level domain). To configure the DNS settings for this domain, u
se the Cloudflare Dashboard."

**Comment by NewFuture** [comment]
买个域名 😄
